client/core/trade.go

```diff
@@ -794,7 +810,11 @@ func (t *trackedTrade) redeemMatches(matches []*matchTracker) error {
 
 	// Send the transaction.
 	redeemWallet, redeemAsset := t.wallets.toWallet, t.wallets.toAsset // this is our redeem
-	coinIDs, outCoin, err := redeemWallet.Redeem(redemptions, redeemAsset)
+	feeRate, err := redeemWallet.FeeRate()
+	if err != nil {
+		feeRate = redeemAsset.MaxFeeRate // or redeemWallet.Info().DefaultFeeRate or a value from a server message?
+	}
+	coinIDs, outCoin, err := redeemWallet.Redeem(redemptions, feeRate)
```

Comment:
That's an option.  I think because I'm not yet certain Redeem won't want to use a server-based fee or what.  I'm OK with having this be done internally at this point though.